dat/missions/zalek/neburesearch/neburesearch_00.lua

```diff
@@ -0,0 +1,221 @@
+text[7] = _([["Oh, hello again, Captain! I'm done just by now here, so we can take off whenever you're ready. I have to calibrate the sensors during the flight so there is no need to rush. Our first destination is %s."
+    After asking what he has done to your ship he just answers, "Why, I just installed the sensors as appointed. It should have no unwanted side effects on your ship."]])
+text[8] = _([["Just a complete mess? Haven't you noticed the color coding? Well, I have my own system so it may appears confusing to you. Don't worry, I know exactly what I'm doing!"
+    His last words are supposed to be reassuring but instead you realize accepting this mission was not the best idea...]])
+text[9] = _([[After reaching the %s system the student enters your cockpit. As he realized he forgot to even tell you his name he introduces himself as Robert Hofer and claims to be a student of professor Voges himself. You haven't ever heard of that name but it sounds like he is famous within House Za'lek.
+    "I will now start with the measurements. The density of the nebula is lower in this sector so that it is less volatile. For the real measurements we have to enter %s. I will tell you once we're done here."]])
+text[10] = _([[Suddenly you lose control over your ship. Apparently most core systems were shut down. Something drains your ship's energy and there are black outs in several parts of your ship.
+    You realize that your shields are down as well. In an environment like this... That's it, your going to die here! You knew accepting this mission was a mistake from the very first moment.]])
+text[11] = _([[It seems you are still alive and all core components are back online!
+    There is a certain someone you should have a serious talk to.]])
+text[12] = _([["Sorry for causing trouble. I'm not quite familiar with the electronics of this ship type. You really should fly a Za'lek ship instead. It wouldn't have possibly happened!"
+    After telling him that your ship was almost destroyed he just replies "That's interesting! So the nebula is corrosive as well? I should investigate the damage it caused to the armour once we landed. But first bring us to the %s system! Don't worry the blackout will not occur again!"
+    It would probably be the best to just drop him off on the next planet...]])
+text[13] = _([["Thank you for your trust! I will try to make it quick. No, wait, I'll rather try to make it without another blackout. Haha!"
+    You start to think it was a big mistake to come to the %s system. "Anyway, just fly a circle or something. I will tell you once the scan is complete."]])
+text[14] = _([["And that's it! Now, let's head back to %s."
+    Meanwhile while looking out of your ship's window you think you saw something passing by, like a ship - but no one you've ever seen before. Nothing showed up on your radar although the object passed by very close. You could ask the student about it.]])
```

Comment:
"no one" -> "not one". ("No one" always refers to people, not objects. I assume this is a simple typo.)

🕵️